python/packages/sdk-schema/.gitignore

```diff
@@ -1 +1 @@
-sdk_schema/schema
+serverless_sdk_schema/schema
```

Comment:
Those changes seem unrelated to this work, can they be proposed with different PR?

Reply:
Removed